Bottom: b8538c94bdd141fcd437e46be51ab9f5bf5c84f8
Top:    312e328820c2f9d8b5ca9eee07b9519546877640
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-09 19:58:56 +0000

Refresh of hid-common-provide-a-header-fi-0

---

diff --git a/src/hid/lpr/lpr.c b/src/hid/lpr/lpr.c
index 01b8119..ac3e2db 100644
--- a/src/hid/lpr/lpr.c
+++ b/src/hid/lpr/lpr.c
@@ -17,7 +17,7 @@
 #include "hid.h"
 #include "../hidint.h"
 #include "../ps/ps.h"
-#include "hid/common/hidint.h"
+#include "hid/common/hidinit.h"
 
 #ifdef HAVE_LIBDMALLOC
 #include <dmalloc.h>
